pkg/stdlib/html/lib.go

```diff
@@ -105,3 +105,43 @@ func waitTimeout(ctx context.Context, value values.Int) (context.Context, contex
 		time.Duration(value)*time.Millisecond,
 	)
 }
+
+func defaultScrollOptions() drivers.ScrollOptions {
+	return drivers.ScrollOptions{
+		Behavior: drivers.ScrollBehaviorInstant,
```

Comment:
We can set int value of `ScrollBehaviorInstant` to 0 instead 2. And do the same with `ScrollHorizontalAlignmentCenter` and `ScrollVerticalAlignmentCenter`.

As a result, method `defaultScrollOptions` will not be needed, an empty structure will already be the default value.

Reply:
I set to default values as they are in DOM spec, but I guess you are right.